drivers/flash/spi_nor.c

```diff
@@ -540,6 +540,45 @@ static int spi_nor_wrsr(const struct device *dev,
 	return ret;
 }
 
+#define MX25R_HIGH_PERF_MODE BIT(1)
```

Comment:
This can go into `#ifdef CONFIG_SPI_NOR_MX_HIGH_PERF_MODE`

Reply:
You could also use `enum { MX25R_HIGH_PERF_MODE = BIT(1) };` inside `mx_specific_enable_high_perf_mode()` instead, as this value is to be used only in this function.